throw "Error came from " + self.name + " module.";
